package.json

```diff
@@ -32,14 +32,16 @@
     "fast-json-stable-stringify": "2.0.0",
     "file-saver": "^1.3.8",
     "fuse.js": "^3.2.0",
+    "http-status-codes": "^1.3.2",
     "immer": "3.1.3",
     "immutable": "^3.8.2",
     "inline-css": "^2.4.1",
-    "lodash": "^4.17.11",
+    "lodash": "^4.17.15",
     "markdown-it": "^8.4.2",
     "moment": "^2.22.2",
     "node-zip": "^1.1.1",
     "office-ui-fabric-react": "6.185.0",
+    "query-string": "^6.8.1",
```

Comment:
Hmm, I also needed to do work with querystrings what did you need it for?

Reply:
My bad. Was originally going to use query string for flighting but moved to setting page.  Want me to leave it in for you?

Reply:
To late now it's already merged, might as well leave it